tcelery/redis.py

```diff
@@ -17,9 +17,9 @@ def unsubscribe_channel(self, channel_name):
     def on_message(self, msg):
         if not msg:
             return
-        if msg.kind == 'message' and msg.body:
+        if msg.kind.decode('ascii') == 'message' and msg.body:
             # Get the list of subscribers for this channel
-            for subscriber in self.subscribers[msg.channel].keys():
+            for subscriber in self.subscribers[msg.channel.decode('ascii')].keys():
```

Comment:
@mher because these UUIDs are surely ascii-encoded and ``decode('utf8')`` will look like result of careless pasting decode/encode everywhere in desperate attempts to "make this stuff work"